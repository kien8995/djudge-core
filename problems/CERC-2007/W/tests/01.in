9 1
3+x/6
17/8-x/8
10 1
(x+10)
10/(x+10)
100 99.9
100
99
10 0.5
1+2*x
2*x
12 1
100
0.5
20 1.5
50+0/5-1/5+1/5
((((((((((((((((((((x))))))))))))))))))))
20 2
(x*x+x)/80+x/10+20
1/(2+x*x/60)
0.1 0.05
4
1
0.5 0.05
4
1
0 0
