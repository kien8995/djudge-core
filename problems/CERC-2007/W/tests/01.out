Pour 0.030 litres / 3.365 cm of water.
Pour 0.878 litres / 2.193 cm of water.
Pour 0.000 litres / 0.000 cm of water.
Pour 0.022 litres / 6.883 cm of water.
Pour 2.854 litres / 0.092 cm of water.
Pour 50.504 litres / 8.117 cm of water.
Pour 0.861 litres / 0.695 cm of water.
Pour 0.000 litres / 0.000 cm of water.
Pour 0.005 litres / 0.162 cm of water.
